hack/boilerplate.go.txt

```diff
@@ -1,13 +1 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
\ No newline at end of file
+
```

Comment:
Why is licensing text being removed?

Reply:
What is the "code-generator", do you mean `controller-gen`?

Reply:
Let's not make this empty, if you need an empty text file please add a `hack/blank.txt` file

Reply:
> What is the "code-generator", do you mean `controller-gen`?

k8s.io/code-generator@v0.21.5, which is actually under kubernetes repo

Reply:
Yes, added hack/custom-boilerplate.go.txt